core/src/main/resources/jenkins/management/AdministrativeMonitorsDecorator/resources.css

```diff
@@ -29,7 +29,6 @@
     height: auto;
     z-index: 0;
     padding: 2em;
-    border: 1px solid #aa;
```

Comment:
Well, `#aaa` looks weird, and the proposed change just removes the broken rule that has no effect on presentation (at least in Firefox).

I consider changing the presentation to be out of scope of this PR.